### PR TITLE
update FAQ 

### PR DESCRIPTION
**Pull request by ManonLef**
## Because

A few FAQ sections in this document are outdated and not in sync with the current Discord status

## This PR

- update information about Thinkful to reflect the latest QA on Discord as edited in April
- add a link to the guide on how to use modmail as added in the `#contact-moderators` channel on discord
- remove help vampire reference and link
- update link on asking questions to lead to the TOP article
- update bootcamp info to reflect current QA on Discord
- 2023-05-23 check and update links 

## Issue

Closes #XXXXX

## Additional Information

If this PR is merged the following needs to be done:
- [ ] Confirm the discord handles are correctly formatted for posting through Dyno/Thor i.e. `<@575252669443211264>` 
- [ ] Update the discord FAQs with Dyno/Thor

If anyone has experience with the above and has the Dyno perms required to do so AND is willing to pick this up, please make yourself known :)

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -93,17 +93,13 @@ Very, but that doesn't mean you can't do it. Learning programming in general is
...
-Help vampires are very common but also very detrimental to online communities. This [article on recognizing help vampire behavior](http://slash7.com/2006/12/22/vampires/) is a good read. Simply asking questions does not make someone a help vampire. 
-
 **How can I help others solve their coding problems?** 
 
 Read our [guide on helping others solve coding problems](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/join-the-odin-community#how-to-help-others-solve-coding-problems). 
 
 **How should I ask questions?**
 
-[Read this]([https://discord.com/channels/505093832157691914/505093832157691916/853784735292325898](https://www.theodinproject.com/how_to_ask)).
+[Read this](https://www.theodinproject.com/how_to_ask).
```
Comment (rlmoser99) [inline FAQ.md:102]: This link's path has been updated to: https://www.theodinproject.com/guides/community/how_to_ask

Comment (ManonLef) [reply FAQ.md]: Thanks for catching that @rlmoser99 
I noticed more links had to be changed so I edited those as well.

**Review by rlmoser99 — APPROVED** [review]
Looks good to me.

**Comment by ManonLef** [comment]
@rlmoser99 just to be sure; am I allowed to merge it then?

Would you be willing to:
- confirm the discord modmail handle format is correct for dyno bot and,
- update the FAQ through dyno once merged?

**Comment by rlmoser99** [comment]
@ManonLef Yes, you are good to merge this.

I have updated the links, but I can't update the text about Modmail, because that block of text is at it's max character limit, so I'll need to move text from one message to another, which takes a bit more focus than I have time for right now. But I should have time to do this after work.

**Comment by rlmoser99** [comment]
@ManonLef Just to follow up and let you know these changes have all been made in Dyno

**Comment by ManonLef** [comment]
> @ManonLef Just to follow up and let you know these changes have all been made in Dyno

Thank you @rlmoser99 

I do notice other sections and rewording are not in sync yet with the FAQ on discord. In the previous PR, we also changed some link formatting and added a link to the getting hired section for example. 

Is there a way I could format this document to make it easier for you to re-post it with dyno to the #FAQ channel instead of having to update each change manually? 

@01zulfi also proposed using our Odin bot to fetch the md file and post it in the relevant channels. Perhaps that is something we could work on? Especially if it can be made so that it will update automatically or more easily.

**Pull request merged**